radio/src/pulses/pxx_arm.cpp

```diff
@@ -348,21 +359,39 @@ void setupPulsesPXX(uint8_t port)
       }
     }
     else {
-      if (i < sendUpperChannels) {
-        int channel = 8 + g_model.moduleData[port].channelsStart + i;
-        int value = channelOutputs[channel] + 2*PPM_CH_CENTER(channel) - 2*PPM_CENTER;
-        pulseValue = limit(2049, (value * 512 / 682) + 3072, 4094);
-      }
-      else if (i < NUM_CHANNELS(port)) {
-        int channel = g_model.moduleData[port].channelsStart + i;
-        int value = channelOutputs[channel] + 2*PPM_CH_CENTER(channel) - 2*PPM_CENTER;
-        pulseValue = limit(1, (value * 512 / 682) + 1024, 2046);
-      }
-      else {
-        pulseValue = 1024;
-      }
+        if(g_model.moduleData[port].channelsCount)
```

Comment:
This will also be true if channelsCount is -1 (=7 channels), is the idea here to have check for > 8 channels?